source/precalculus/source/04-PR/01.ptx

```diff
@@ -526,7 +532,14 @@
               <li> <p> upwards   </p> </li>
               <li> <p> <m>(-1,-4)</m>; minimum   </p> </li>
               <li> <p> <m>x=-1</m>   </p> </li>
-              <li> <p> (placeholder for graph)   </p> </li>
+              <li> <p> <figure width="100%" xml:id="graph-of-x^2+2*x-3">
```

Comment:
```suggestion
              <li> <p> <figure width="100%" xml:id="graph-of-x2+2x-3">
```
I don't think ^ or * are allowed in xml:ids, which was causing the build to fail.